app/src/main/kotlin/org/gnucash/android/model/Recurrence.kt

```diff
@@ -0,0 +1,437 @@
+     * The multiplier for the period type. The default multiplier is 1.
+     * e.g. bi-weekly actions have period type [PeriodType.WEEK] and multiplier 2.
+     */
+    var multiplier = 1 //multiplier for the period type
+
+    init {
+        this.periodType = periodType
+        periodStart = Timestamp(System.currentTimeMillis())
+    }
+
+    /**
+     * Returns an approximate period for this recurrence
+     *
+     * The period is approximate because months do not all have the same number of days,
+     * but that is assumed
+     *
+     * @return Milliseconds since Epoch representing the period
+     */
+    @get:Deprecated("Do not use in new code. Uses fixed period values for months and years (which have variable units of time)")
+    val period: Long
+        get() {
+            var baseMillis: Long = 0
+            when (periodType) {
+                PeriodType.HOUR -> baseMillis = RecurrenceParser.HOUR_MILLIS
+                PeriodType.DAY -> baseMillis = RecurrenceParser.DAY_MILLIS
+                PeriodType.WEEK -> baseMillis = RecurrenceParser.WEEK_MILLIS
+                PeriodType.MONTH -> baseMillis = RecurrenceParser.MONTH_MILLIS
+                PeriodType.YEAR -> baseMillis = RecurrenceParser.YEAR_MILLIS
+            }
+            return multiplier * baseMillis
+        }
+
+    /**
+     * Returns the event schedule (start, end and recurrence)
+     *
+     * @return String description of repeat schedule
+     */
+    val repeatString: String
+        get() {
+            val repeatBuilder = StringBuilder(frequencyRepeatString)
+            val context = GnuCashApplication.getAppContext()
+            val dayOfWeek = SimpleDateFormat("EEEE", GnuCashApplication.getDefaultLocale())
+                .format(Date(periodStart.time))
+            if (periodType === PeriodType.WEEK) {
+                repeatBuilder.append(" ")
+                    .append(context.getString(R.string.repeat_on_weekday, dayOfWeek))
+            }
+            if (periodEnd != null) {
+                val endDateString = SimpleDateFormat.getDateInstance().format(
+                    Date(
+                        periodEnd!!.time
+                    )
+                )
+                repeatBuilder.append(", ")
+                    .append(context.getString(R.string.repeat_until_date, endDateString))
+            }
+            return repeatBuilder.toString()
+        }
+
+//        =======================================================================
+//This section complies with the formal rules, but the betterpickers library doesn't like/need it
+//        SimpleDateFormat startDateFormat = new SimpleDateFormat("'TZID'=zzzz':'yyyyMMdd'T'HHmmss", Locale.US);
+//        ruleBuilder.append("DTSTART;");
+//        ruleBuilder.append(startDateFormat.format(new Date(mStartDate)));
+//            ruleBuilder.append("\n");
+//        ruleBuilder.append("RRULE:");
+//        ========================================================================
```

Comment:
duplicate commented out code block (see below L144-L151)

Reply:
Sorry! I didn't even noticed those. They should be gone now.